mobile/src/main/java/org/hawkular/client/android/activity/DetailActivity.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.client.android.activity;
+
+import org.hawkular.client.android.R;
+import org.hawkular.client.android.backend.model.Alert;
+import org.hawkular.client.android.util.Formatter;
+import org.hawkular.client.android.util.Fragments;
+import org.hawkular.client.android.util.Intents;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Detail activity.
+ */
+public abstract class DetailActivity extends AppCompatActivity {
+
+    @BindView(R.id.toolbar)
+    Toolbar toolbar;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_container);
+
+        setUpBindings();
+
+        setUpToolbar();
+
+        setUpDetails();
```

Comment:
@danielpassos Will do that tomorrow for sure. Right now a kid is stuck because of this. 

So can you please let it pass for a day.

Just a day ;)